src/treectl.c

```diff
@@ -2704,8 +2704,8 @@ TreeControlWndProc(
 
       UpdateWindow(hwndStatus);
 
-      iSelHilite = ((LPDROPSTRUCT)lParam)->dwControlData;
-      RectTreeItem(hwndLB, iSelHilite, (BOOL)wParam);
+      iSelHighlight = ((LPDROPSTRUCT)lParam)->dwControlData;

```

Comment:
I recognize that `Hilite` is used as an alternate spelling of `Highlight`, but `Hilight` is definitely not, so I'm suggesting standardizing on the long form (which also exists in the repo).

Reply:
Hilight is a logical shortening of Highlight (aka Hi~~gh~~light or hi-light)